README.md

```diff
@@ -40,41 +37,51 @@ and currently does not support version 3.8 and above. It is recommended to insta
 
 To set up on your local machine:
 
-To install core utilities, CPU-based algorithms, and dependencies:
+* To install core utilities, CPU-based algorithms, and dependencies:
+
+    1. Ensure software required for compilation and Python libraries
+       is installed.
+
+       + On Linux this can be supported by adding:
+
+         ```bash
+         sudo apt-get install -y build-essential libpython<version>
+         ``` 
+
+         where `<version>` should be `3.6` or `3.7` as appropriate.
```

Comment:
Dose it support Python 3.8 now? To support Databricks 7.3+, Python 3.8 is required.

Reply:
3.8 may also need to be added to:
https://github.com/microsoft/recommenders/blob/f4636b2e8e74d87233fd53958c5e0526674fc7f6/setup.py#L127-L134

Reply:
that's a good point, if my memory doesn't failed me, we had some problems with 3.8 and numpy? @anargyri might know more when he is back

Reply:
The unit tests work with Python 3.8. See https://github.com/microsoft/recommenders/pull/1617